### PR TITLE
Fix left nav padding on VAMC pages

### PR DESCRIPTION
**Pull request by mpelzsherman**
See https://github.com/department-of-veterans-affairs/va.gov-team/issues/9945

### Mobile
VA_Pittsburgh_Health_Care___Veterans_Affairs (1)

### Desktop
VA_Pittsburgh_Health_Care___Veterans_Affairs

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by brangi — APPROVED** [review]

**Comment by mpelzsherman** [comment]
Also added the side nav code to the facilities team in .github/CODEOWNERS.

**Comment by U-DON** [comment]
For graphical changes like these, we typically expect a screenshot.

Code owners also should not be combined with a feature change.

**Comment by mpelzsherman** [comment]
Screen shots added (and comps were also available in the original ticket).

I will open a separate PR for the coder owner changes.

**Comment by mpelzsherman** [comment]
Opened #14094 for codeowners changes.

**Review by U-DON — APPROVED** [review]
LGTM, but trying to get the build to cooperate.

File: src/platform/site-wide/sass/modules/_m-side-nav.scss [hunk 1]
```diff
@@ -22,17 +22,31 @@ $side-nav-color-dark: #323a45;
     cursor: pointer;
     width: calc(100% - 45px);
     margin-left: 22px;
+
+    &::after {
+      content: "\A";
```
Comment (ncksllvn) [inline src/platform/site-wide/sass/modules/_m-side-nav.scss:27]: Pardon my ignorance, but what does `"\A"` do? I'm assuming that's not rendering an "A"

**Review by ncksllvn — APPROVED** [review]

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

File: src/platform/site-wide/sass/modules/_m-side-nav.scss (hunk 1 above)
Comment (mpelzsherman) [reply src/platform/site-wide/sass/modules/_m-side-nav.scss:27]: Adds a line break without having to add a `<br>`. See https://stackoverflow.com/questions/34060665/css-content-character-a-takes-more-space-than-br-tag/34060752

**Pull request merged**